core/scss/components/brand-bar/_brand-bar.scss

```diff
@@ -0,0 +1,32 @@
+@charset "UTF-8";
+
+//
+// Brand Bar
+//
+// Stanford brand bar with wordmark logo. This sits at the very top of a website.
+//
+// .su-brand-bar--default - Default brand bar with white wordmark on cardinal red background
+// .su-brand-bar--bright - White wordmark on bright red background
+// .su-brand-bar--dark - White wordmark on dark grey background
+// .su-brand-bar--white - cardinal red wordmark on white background
+//
+// Markup: ../../templates/components/brand-bar/brand-bar.twig
+//
+// Style guide: Components.Brand Bar
+//
+
+#su-brand-bar,
+.su-brand-bar--default {
+
+  @include centered-grid-container;
+  width: 100%;
```

Comment:
We don't want our components generating markup if they don't have to. We also want to make our mixins available to those using the `decanter-no-markup` file. So, I'm thinking we should package the code up into a mixin and call the mixin at the class level.

Reply:
I could rewrite it so it's similar to how I wrote the CTA mixin.

Reply:
Disregarding this comment after discussion with Joe and Shea.

Reply:
Components are allowed to generate markup.

Reply:
Yep! Thanks, all, for the discussion on this earlier in the week. Makes sense.